scio-core/src/main/scala/com/spotify/scio/testing/JobTest.scala

```diff
@@ -98,11 +98,21 @@ object JobTest {
       this.copy(distCaches = this.distCaches + (key -> value))
     }
 
-    def run(): Unit = {
-      val testId = newTestId(className)
+    def setup(testId: String): Unit = {
```

Comment:
Name it `setUp` just to be consistent with JUnit, etc. And remove the `testId` arg.